src/main/java/com/globant/carrito/cart/Carts.java

```diff
@@ -0,0 +1,134 @@
+package com.globant.carrito.cart;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Date;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import com.globant.carrito.product.Items;
+
+
+@Entity
+public class Carts {
+	
+	@Id
+	@GeneratedValue
+	private int cartId;
+	
+	@ManyToOne(cascade = CascadeType.ALL)
+	private int clientId;
+	
+	@OneToOne(cascade = CascadeType.ALL)

```

Comment:
ah, lo pensé que como es una colección de items, con OneToOne era suficiente. Pero viendo lo que mencionas, la relación es independiente de dónde voy a almacenar los items dentro de cart. Ahora lo cambio.

Date: Wed, 4 Feb 2015 06:54:09 -0800
From: notifications@github.com
To: cart_client@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [cart_client] Avance del shopping cart hasta el 03/02/15. (#1)

In src/main/java/com/globant/carrito/cart/Carts.java:

> +import javax.persistence.OneToOne;
> +
> +import com.globant.carrito.product.Items;
> +
> +
> +@Entity
> +public class Carts {
> +	
> +	@Id
> +	@GeneratedValue
> +	private int cartId;
> +	
> +	@ManyToOne(cascade = CascadeType.ALL)
> +	private int clientId;
> +	
> +	@OneToOne(cascade = CascadeType.ALL)

Acá sería @OneToMany


—
Reply to this email directly or view it on GitHub.